src/commands/deploy.ts

```diff
@@ -209,18 +209,25 @@ export default class Deploy extends Command {
             `    ${`http://${config.app}.liara.localhost`}`
           : `    ${`https://${config.app}${defaultSubdomain}`}`;
 
-        const { domains } = await this.got(
+        let { domains } = await this.got(
           `v1/domains?project=${config.app}`
         ).json<IGetDomainsResponse>();
 
         if (!domains.length || project.defaultSubdomain)
           this.log(urlLogMessage);
 
+        domains = domains.reverse();
```

Comment:
Please use `slice` instead of an incremental variable:

```suggestion
        const firstFiveDomains = domains.reverse().slice(0, 5)
```